### PR TITLE
fix imread import in example

### LINKED ISSUE #2095: scipy.misc.imread() replaced by scipy.imageio.imread() in v1.2
`scipy.misc.imread()` was deprecatd in SciPy 1.0 & replaced in SciPy 1.2 by `scipy.imageio.imread()`

https://docs.scipy.org/doc/scipy-1.2.1/reference/generated/scipy.misc.imread.html

This is causing failures in CircleCI. 

I will work on this once PR  #2076 doctest problem has been addressed, since we need this issue to be resolved before it can be merged. Intended today.

### PR DESCRIPTION
**Pull request by jeromedockes**
fixes #2095

### CODE REVIEW

**Comment by kchawla-pi** [comment]
I had assigned this one to myself.

**Comment by jeromedockes** [comment]
> I had assigned this one to myself.

sorry didn't see that

**Comment by kchawla-pi** [comment]
Oh don't close it! Now you have done the work.

**Comment by kchawla-pi** [comment]
Should we do a try-except here? The minimum supported scipy is 0.19, which means the examples are expected to work with that as well.

**Comment by jeromedockes** [comment]
> Should we do a try-except here? The minimum supported scipy is 0.19, which means the examples are expected to work with that as well.

I think importing `imread` from pyplot is perfecly fine and works in all cases so let's keep the example simple

File: examples/02_decoding/plot_haxby_stimuli.py [hunk 1]
```diff
@@ -27,8 +26,8 @@
     for i in range(48):
         plt.subplot(6, 8, i + 1)
         try:
-            plt.imshow(imread(file_names[i]), cmap=plt.cm.gray)
-        except:
+            plt.imshow(plt.imread(file_names[i]), cmap=plt.cm.gray)
+        except Exception:
```
Comment (kchawla-pi) [inline examples/02_decoding/plot_haxby_stimuli.py]: We should make it a more specific exception, FileNotFoundError  or something only catching missing file condition.

**Pull request merged**